app/src/main/java/com/spacelobster/fuckingbills/fragment/HelloFragment.kt

```diff
@@ -67,16 +71,6 @@ class HelloFragment : Fragment(), AnkoLogger {
         }
     }
 
-    private fun scaleHouseDown() {
```

Comment:
Let's decide to do not put private methods before @override. I like when in the classes things are placed as vars->constructors->overrides->public/private methods(public before private preferable)